numpy/core/src/umath/loops_umath_fp.dispatch.c.src

```diff
@@ -1,14 +1,30 @@
 /*@targets
- ** $maxopt baseline avx512_skx
+ ** $maxopt baseline
+ ** avx512_skx
+ ** asimd
```

Comment:
I think `advsimd` is the preferred name.

Reply:
This is the standard naming within `numpy` itself, I don't think there's a strong enough reason to change it?